### DISCUSSION TITLE
What is the default Lifecycle Configuration for a bucket?

### DISCUSSION THREAD

**Discussion by toaksoy** [minio/minio#20585] [Q&A] [answered]
While running: 
```
aws --endpoint-url $MINIO_ENDPOINT_URL s3api get-bucket-lifecycle-configuration --bucket my-bucket
```
I am facing:
_An error occurred (NoSuchLifecycleConfiguration) when calling the GetBucketLifecycleConfiguration operation: The lifecycle configuration does not exist_

So, I wonder when the objects in my-bucket would expire and get deleted, is there a default retention period? What's the default lifecycle configuration if there is none set explicitly?

**Comment by klauspost** [comment, marked as answer]
Nothing. Nothing happens unless you ask for it.
